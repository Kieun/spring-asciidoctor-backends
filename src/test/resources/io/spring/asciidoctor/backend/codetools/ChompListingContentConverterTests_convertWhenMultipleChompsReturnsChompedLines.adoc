= Test Document
:chomp_package_replacement: com.example

[source,java,chomp="all"]
----
/*
 * Copyright 2014-2020 the original author or authors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package com.example.some.subpackage;

public class ChompExample {

	private final Thing thing;

	private final OtherThing otherThing;

	// @formatter:off
	public ChompExample() {
		this.thing = /* @chomp:line ... your thing */ new DocumentedThing
		this.otherThing = /**/ new OtherThing();
	}
	// @formatter:on

}
// @chomp:file

class DocumentedThing extends Thing {
}

----
