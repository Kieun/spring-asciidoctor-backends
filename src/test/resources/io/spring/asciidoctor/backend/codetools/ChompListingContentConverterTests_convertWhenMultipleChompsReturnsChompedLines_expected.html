package com.example;

public class ChompExample {

	private final Thing thing;

	private final OtherThing otherThing;

	public ChompExample() {
		this.thing = ... your thing
		this.otherThing = ...
	}

}